Process.md

```diff
@@ -0,0 +1,55 @@
+# OpenCensus change process
+
+This document outlines a process for proposing, approving and implementing changes in OpenCensus.
+
+Status: WIP, improvements/suggestions welcome.
+
+## Scope
+
+This change process covers changes to the OpenCensus Specs. Generally, all new APIs or major
+API changes should be documented in OpenCensus Specs. This is to ensure consistency across
+language implementations, as far as practical.
+
+This document does not cover improvements to individual OpenCensus libraries that don't
+require Spec changes. These should be filed as issues on the individual OpenCensus library
+repository.
+
+## Process
+
+### Step 1: Opening an issue
+
+Open an issue on the (opencensus-specs repo)[https://github.com/census-instrumentation/opencensus-specs/issues/new].
+
+The issue should describe the problem you're trying to solve and the proposed solution (if any).
+
+When opened, the label "proposal" should be applied. The issue will move out of "proposal" state when it is
```

Comment:
Do you want to clarify the purpose of "proposal" label? should people not work on something unless it gets to approved state?

Reply:
Yes, will do